lib/display/index.ts

```diff
@@ -66,7 +66,11 @@ export async function display(
   if (!hasDependencies) {
     exitWith(
       ExitCode.NoSupportedFiles,
-      `${output}\nCould not detect supported target files in ${options?.path}`,
+      `${output}\nCould not detect supported target files in ${
+        options?.path
```

Comment:
do we want to add a default value if it's not defined? (instead of empty value?)

Reply:
Think we should be ok - we should have exited if options.path is false (lib/scan.ts line 34).